third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -55,6 +60,63 @@ WinUWPH264EncoderImpl::~WinUWPH264EncoderImpl() {
   Release();
 }
 
+namespace {
+HRESULT MakeMediaTypeOut(UINT32 width,
+                         UINT32 height,
+                         UINT32 target_bps,
+                         UINT32 frame_rate,
+                         IMFMediaType** res_out) {
+  HRESULT hr = S_OK;
+  // output media type (h264)
+  ON_SUCCEEDED(MFCreateMediaType(res_out));
+  auto media_type_out = *res_out;
+  ON_SUCCEEDED(media_type_out->SetGUID(MF_MT_MAJOR_TYPE, MFMediaType_Video));
+  ON_SUCCEEDED(media_type_out->SetGUID(MF_MT_SUBTYPE, MFVideoFormat_H264));
+  // Lumia 635 and Lumia 1520 Windows phones don't work well
+  // with constrained baseline profile.
+  // todo(fibann): this is obsolete and should probably use the profile passed
```

Comment:
>todo(fibann): [](start = 5, length = 13)

Please file a task and remove this comment.

Reply:
Logged as https://github.com/webrtc-uwp/webrtc-uwp-sdk/issues/207